app/pb_kits/playbook/pb_icon_body_title/docs/_icon_body_title_default.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+import IconBodyTitle from "../_icon_body_title.jsx"
+
+function IconBodyTitleDefault() {
+  return (
+    <div>
+      <IconBodyTitle icon="user" title="Nitro" body="Virtual Assistant" link="#" />
```

Comment:
Just minor eslint code style nits (alphabetize + formatting) for readability here

```jsx
<IconBodyTitle
    body="Virtual Assistant"
    icon="user"
    link="#"
    title="Nitro"
/>
```